documentation/document-api-guide.html

```diff
@@ -179,10 +238,10 @@ <h2 id="document-updates">Document updates</h2>
         new IntegerFieldValue(1), // array index
         new AssignValueUpdate(new IntegerFieldValue(2))));  // value at index
 
-// Composite - update weightedset element
-upd.addFieldUpdate(FieldUpdate.createMap(type.getField("aWeightedset"),
+// DO NOT USE - createMap can cause crash / missing updates TODO FIXME
```

Comment:
Hopefully we'll remember to update this once the bug is fixed 😉

Reply:
I'll consider making it a publish issue for easier reference